version/version.go

```diff
@@ -0,0 +1,34 @@
+package version
+
+// Copyright 2017 Microsoft Corporation
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+import (
+	"fmt"
+	"runtime"
+)
+
+// Number contains the semantic version of this SDK.
+const Number = "v10.14.0"
+
+var (
+	// UserAgent builds a string containing the Go version, system archityecture and OS,
+	// and the go-autorest version.
+	UserAgent = fmt.Sprintf("Go/%s (%s-%s) go-autorest/%s",
```

Comment:
Hmm I'm thinking that this should not be exported and instead expose the value via a func.

Reply:
so people can't change it ?

Reply:
Yeah.  We already provide a facility for adding to the user-agent but we probably don't want folks to have the ability to muck about with the default info.